src/pages/komodo-defi-framework/api/v20/enable_eth_with_tokens/index.mdx

```diff
@@ -67,7 +67,6 @@ Additionally, it supports ERC20 tokens on the ETH chain and associated ERC20 lik
           "gui_auth": true
         }
       ],
-      "rpc_mode": "Http",
```

Comment:
This section is the "Request including optional gas station parameters" section. Should we not still display the optional parameter with its value set to 'default' instead?

Reply:
As it defaults to `Default`, seems a waste to define what's defined by default. Might be better to add an example with the non-default value.